display/ssd1306.h

```diff
@@ -309,18 +322,18 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
     t_ = 0;
     if (SaberBase::IsOn()) {
       if (SaberBase::Lockup() && IMG_lock && !ignore_lockup) {
-	SetFile(&IMG_lock, 3600000.0);
-      } else if (looped_on_ == Tristate::True) {
-	SetFile(&IMG_on, font_config.ProffieOSOnImageDuration);
+        SetFile(&IMG_lock, 3600000.0);
+      } else if (looped_on_ != Tristate::False) {
```

Comment:
This seems like a major change.
Why are we changing it?
Are you sure it's right?

Reply:
Well I certainly didn't do it on my own. This was from a conversation with you about something with idle.bmp I believe. I took a look back through the commit history for ssd1306, through closed PRs, PMs, The Crucible, and TRA. 
I can not find the conversation.
I do not pretend to understand what this does, but if I recall, Tristate was added as a major rewrite by you to get idle.bmp, and the Mass Storage stuff worked in correctly.
Unless I changed it completely randomly, I don't see any other reason for the change other than you said to try that and it fixed something.

Reply:
Thinking about it, it seem reasonable.
If we get here and we don't know if on is looping yet, then we should probably play it to find out, which is what this change does.

Reply:
works with a looped image and single frame (plays for FontImageDuration), and idle.bmp works, so this seems correct.